components/autocalc/autocalc.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef COMPONENTS_AUTOCALC_AUTOCALC_H
+#define COMPONENTS_AUTOCALC_AUTOCALC_H
+
+#include <string>
+
+#include "store.hpp"
+
+namespace ESM
+{
+    struct NPC;
+    struct Race;
+    struct Class;
+}
+
+namespace AutoCalc
+{
+    // wrapper class for sharing the autocalc code between OpenMW and OpenCS
+    class  StatsBase
+    {
+
+    public:
+
+        StatsBase();
+        virtual ~StatsBase();
+
+        virtual unsigned char getBaseAttribute(int index) const = 0;
+
+        virtual void setAttribute(int index, unsigned char value) = 0;
+
+        virtual void addSpells(std::string id) = 0;
```

Comment:
Again, the code was simply copied over, keeping the names of the methods the same.